src/GitLab/Types.hs

```diff
@@ -244,6 +248,30 @@ data RepositoryTree = RepositoryTree
   , repositoryTreeMode :: Text
   } deriving Show
 
+data CommitActionType = ActionCreate
+                      | ActionDelete
+                      | ActionMove
+                      | ActionUpdate
+                      deriving Show
+
+data CommitActionEncoding = EncodingText
+                          | EncodingBase64
+                          deriving Show
+
+data CommitAction = CommitAction 
+  { commitActionAction       :: CommitActionType
+  , commitActionFilePath     :: FilePath
+  , commitActionContent      :: Maybe Text
+  , commitActionPreviousPath :: Maybe FilePath
+  , commitActionEncoding     :: Maybe CommitActionEncoding
```

Comment:
Same applies to vertical alignment with colons here. We have to re-align all these fields once we added a field that has longer name. Could we stop the alignment?